holidays/countries/bonaire_sint_eustatius_and_saba.py

```diff
@@ -0,0 +1,82 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class BonaireSintEustatiusAndSaba(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Bonaire Sint Eustatius And Saba holidays.
+
+    References:
+        * [Public Holidays 2025](https://www.sabagov.nl/residents-visitors/public-holidays)
```

Comment:
I'm a bit sceptical about our two 3rd party sources here, but if you plan to use them, please Internet Archive'd them as well 👍 
```suggestion
        * [Public Holidays 2025](https://web.archive.org/web/20250620050728/https://english.rijksdienstcn.com/social-affairs-work/employment/official-holidays)
        * [Public Holidays 2025 in Saba](https://web.archive.org/web/20250620050723/https://www.sabagov.nl/residents-visitors/public-holidays)
```

There's at least one `StaticHolidays` instance here: [Bonaire declaring MAY, 2 a Bridge Holiday for 2025](https://bonairegov.com/nieuwsoverzicht/artikel/vrijdag-2-mei-2025-aangewezen-als-vrije-dag-voor-ambtenaren)

Reply:
Consolidated into newer code review comment 🙏